test152/font.cpp

```diff
@@ -777,38 +777,4 @@ const unsigned char pic_HARRIS[]= {
     0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
     0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
 };
-const unsigned char pic_XVF[] = {
```

Comment:
This is the new pic_XVF splash image, displayed on boot. You might be able to re-add it in another commit to this branch, and if it worked then the CI pipeline should succeed.

Reply:
Maybe for first time we can block this part of code? And later we will be able to sort out the problem?

Reply:
I think I can edit this pull request, standby

Reply:
Apparently yes! Fixed!